src/components/SearchResults.tsx

```diff
@@ -1,18 +1,55 @@
 import React from 'react';
+import Button from '@material-ui/core/Button';
 import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import LifeEventPicker from './LifeEventPicker';
+import RelationshipPicker from './RelationshipPicker';
+import NamePicker from './NamePicker';
+import SearchResultsTable from './SearchResultsTable';
 import Typography from '@material-ui/core/Typography';
 import { makeStyles } from '@material-ui/core/styles';
+import SearchResultsLocation from './SearchResultsLocations';
+import SearchResultsCategory from './SearchResultsCategory';
 
 export default function SearchResults(): JSX.Element {
   const classes = useStyles();
+
   return (
-    <Container className={classes.paper}>
-      <Typography>Hello World</Typography>
+    <Container component="main" className={classes.container}>
+      <Grid container spacing={2}>
+        <Grid item xs={4}>
+          <form noValidate>
+            <Typography className={classes.paper}>Refine your search</Typography>
+            <NamePicker />
+            <LifeEventPicker />
+            <RelationshipPicker />
+            <SearchResultsLocation />
+            <SearchResultsCategory />
+            <div style={{ marginTop: 50 }}>
+              <label onClick={() => console.log('click')}>
```

Comment:
What purpose does 'label' have here? (I'm actually not that great with HTML and CSS, so I really don't know the answer!) Since the label 'Search' is specified in the Button itself, is label needed? Also, Button has an onClick property...

Reply:
Thanks for pointing out the label! It was from something else that I was trying out and had forgotten to remove. 
I have removed the label and added onClick to Button.